app/src/pages/header/components/AddMovieButton.jsx

```diff
@@ -1,23 +1,23 @@
-import styled from 'styled-components';
-
-const Button = styled.button`
-    border: none;
-    border-radius: 4px;
-    color: #f65261;
-    background: rgba(255, 255, 255, 0.2);     
-    padding: 10px 15px;
-`
-
-const BoldText = styled.span`
-    font-weight: bold;
-`
-
-function AddMovieButton(){
-    return (
-        <Button>
-            <BoldText>+ ADD MOVIE</BoldText>
-        </Button>
-    );
-}
-
+import styled from 'styled-components';
+
+const Button = styled.button`
+    border: none;
+    border-radius: 4px;
+    color: #f65261;
+    background: rgba(255, 255, 255, 0.2);     
+    padding: 10px 15px;
+`
+
+const BoldText = styled.span`
+    font-weight: bold;

```

Comment:
I think this component can be moved to global. There are many places where we can use it

Reply:
I'm not sure that it's good idea to place `bold` styled `span` in separated component)